src/services/ArticleService.js

```diff
@@ -69,6 +86,35 @@ export class ArticleService {
 
 		return {};
 	}
+
+	async createArticle(title) {
+		try {
+			const { data } = await axios.post(`${backendAPI}/${this.BASE_URL}`, {
+				title
+			});
+
+			return data;
+		} catch ({ response }) {
+			throw new Error(response?.data?.message ?? defaultError);
+		}
+	}
+
+	async getArticleRevisions(id) {
+		try {
+			const { data } = await axios.get(`${backendAPI}/${this.BASE_URL}/${id}/revisions`);
+
+			return data.response.map(revision => {
+				const createdDate = new Date(+revision.createdOn).toString().split(" ");
+
+				return {
+					...revision,
+					createdOn: `${createdDate[2]} ${createdDate[1]} ${createdDate[3]}`
+				};
```

Comment:
I assume this should also follow suit and be consistent with the others?

```suggestion
				return {
					...revision,
					createdOn: ArticleService.formatArticleDate(+article.createdOn),
					updatedOn: ArticleService.formatArticleDate(+article.updatedOn)
				};
```